src/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.19
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.0
 	github.com/alcionai/clues v0.0.0-20230314154528-c469e1adafb6
+	github.com/armon/go-metrics v0.4.0
```

Comment:
```
go: found github.com/hashicorp/go-metrics in github.com/hashicorp/go-metrics v0.4.1
go: github.com/alcionai/corso/src/cli imports
        github.com/hashicorp/go-metrics: github.com/hashicorp/go-metrics@v0.4.1: parsing go.mod:
        module declares its path as: github.com/armon/go-metrics
                but was required as: github.com/hashicorp/go-metrics
```

/shrug

Reply:
Weird - and why isn't this `v0.4.1`?

Reply:
This has been fixed recently - https://github.com/hashicorp/go-metrics/pull/143